simphony/cuds/abstractlattice.py

```diff
@@ -22,6 +23,50 @@ class ABCLattice(object):
     """
     __metaclass__ = ABCMeta
 
+    def __init__(self, size, origin, a, b, c, alpha, beta, gamma):
```

Comment:
I would change this to  `def __init__(self, size, origin, primitive_vectors, kind=None)`